patterns/photo-blog-home-template.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: Photo blog home
+ * Slug: twentytwentyfive/photo-blog-home
+ * Template Types: front-page, home
+ * Viewport width: 1400
+ * Inserter: no
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:template-part {"slug":"header","area":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--60)">
+	<!-- wp:heading {"textAlign":"center","level":1,"fontSize":"x-small"} -->
+	<h1 class="wp-block-heading has-text-align-center has-x-small-font-size">Stories</h1>
+	<!-- /wp:heading -->
+
+	<!-- wp:heading {"textAlign":"center","fontSize":"xx-large"} -->
+	<h2 class="wp-block-heading has-text-align-center has-xx-large-font-size">Tell your story</h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:pattern {"slug":"twentytwentyfive/photo-blog-posts"} /-->
```

Comment:
I am not able to reproduce this, can you write what steps you took to get to this result?
Are you sure there is no caching breaking any updates?

Reply:
@carolinan here's what I did:

- Open the site editor.
- Go to the home blog template.
- Pick the new design from the design section.
- See the design applied and view it on the front end.

I'm using Firefox on a viewport bigger than `1440px`. I also checked Chrome, and I see the same results regarding spacing and the content taking the full width instead of the wide width. 

This could be similar to what we fixed [here](https://github.com/WordPress/twentytwentyfive/pull/129). 

What's the viewport size for you when viewing these templates? If it's `1440px` or less, it's likely that you won't see these problems.

Reply:
I can see from the screenshot that the selected element, the UL for the post template, has `alignfull`. That is not part of the current PR.

Settings:
https://github.com/WordPress/twentytwentyfive/pull/123/files#diff-ecab6825b46c8296752891524bf1dd8316a64712982933886d988c9356b98210R27

The group block that is wrapping the post template has layout type default, so the align full option is not even available on the post template block.

Can you share the markup for the post template on the front and also double check that the in the editor, the post template is not set to full width because then the template has still somehow not been reset.

Reply:
I thought so but I have experienced the same and asked in core-editor on Slack.